mix.exs

```diff
@@ -27,6 +28,13 @@ defmodule Craft.MixProject do
     ]
   end
 
+  defp aliases do
+    [
+      "compile.nif":
```

Comment:
can you get this to build the nif automatically when required? (probably need a makefile)